### PR TITLE
OCP-811: Ensure we only send the specified limit in a batch

### PR DESCRIPTION
**Pull request by SoAG**
This change is [Reviewable](https://reviewable.io/reviews/ZaiusInc/app-sdk/137)

### CODE REVIEW

**Review by pawelziebaopti — APPROVED** [review]

File: src/app/lib/tests/Batcher.test.ts [hunk 1]
```diff
@@ -39,5 +39,16 @@ describe('Batcher', () => {
       await batcher.flush();
       expect(fn).not.toHaveBeenCalled();
     });
+
+    it('does not cause the batch limit to be exceeded ' +
```
Comment (pawelziebaopti) [inline src/app/lib/tests/Batcher.test.ts:43]: this doesn't really check the limit. the limit is 5 and it flushes after adding only 2 items.

**Review by pawelziebaopti — CHANGES_REQUESTED** [review]

File: src/app/lib/tests/Batcher.test.ts (hunk 1 above)
Comment (SoAG) [reply src/app/lib/tests/Batcher.test.ts:43]: have another look

**Review by pawelziebaopti — APPROVED** [review]

**Review by pawelziebaopti — CHANGES_REQUESTED** [review]
don't you have to bump the version as well?

**Comment by SoAG** [comment]
done

**Review by pawelziebaopti — APPROVED** [review]

**Pull request merged**